ingestion/src/metadata/ingestion/source/database/dbt_source.py

```diff
@@ -204,21 +208,18 @@ def _parse_data_model_upstream(self, mnode):
         return upstream_nodes
 
     def _parse_data_model_columns(
-        self, model_name: str, mnode: Dict, cnode: Dict
+        self, _: str, mnode: Dict, cnode: Dict
     ) -> List[Column]:
         columns = []
         ccolumns = cnode.get("columns")
```

Comment:
why is that named `ccolumns`?

Reply:
not sure why it's like that but I guess that's pointing to columns of catalogue node

Reply:
will rename this